packages/shared/src/components/Pill.tsx

```diff
@@ -34,10 +24,7 @@ const Pill = ({
     <div
       className={classNames(
         pillSizeToClassName[size],
-        'rounded-10',
-        'bg-theme-overlay-float-cabbage',
-        'px-2',
-        'inline-flex items-center',
+        'rounded-xl p-2 inline-flex items-center self-start',
```

Comment:
I think this should be rounded-10 from the design.
Lee just changes this, but think you reverted it again.

Reply:
You asked to use `-xl` one which is multiple of 4 right?

Reply:
Roger, thus my confusion as well. Will update 👍